collector.go

```diff
@@ -348,17 +348,17 @@ func collectGenericSensor(ch chan<- prometheus.Metric, state float64, data senso
 	)
 }
 
-func (c collector) collectMonitoring(ch chan<- prometheus.Metric, creds Credentials) error {
+func (c collector) collectMonitoring(ch chan<- prometheus.Metric, creds Credentials) int {
 	output, err := ipmiMonitoringOutput(c.target, creds.User, creds.Password)
 	if err != nil {
-		log.Errorln(err)
-		return err
+		log.Errorf("Failed to collect ipmimonitoring data: %s", err)
```

Comment:
If we're going to ignore this error, this should be `log.Debugf()`.

Reply:
True, will change

Reply:
I am trying to guess what would be most intuitive here. I am thinking: if fetching the data data fails, it is probably not supported by the device, we print a debug message and carry on. However, if parsing the data fails, this is probably something that should be looked at and i would consider keeping that message an error message. Same for other tools. Any opinion on that?

Reply:
Yes, that seems like the right thing to do.